src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto/Roboto-Medium.ttf);
+}
+
+html {
+  font-family: "Roboto", serif;
```

Comment:
Consider using a more modern font fallback like 'sans-serif' instead of 'serif' for the Roboto font.

Reply:
"serif" has been changed to "sans-serif"
thx